libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/viewmodel/MapboxNavigationViewModel.kt

```diff
@@ -78,6 +81,31 @@ internal class MapboxNavigationViewModel(
     private val _voiceInstructions: MutableSharedFlow<VoiceInstructions> = MutableSharedFlow()
     val voiceInstructions: Flow<VoiceInstructions> = _voiceInstructions
 
+    // This may be temporary. We need some way to start a trip session to further development.
+    // This is here because this class has a reference to MapboxNavigation
+    @SuppressLint("MissingPermission")
+    fun startTripSession() {
+        mapboxNavigation.startTripSession()
+    }
+
+    // This may be temporary. We need some way to start a trip session to further development.
+    // This is here because this class has a reference to MapboxNavigation
+    @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
+    fun startSimulatedTripSession(location: Location) {
+        val event = ReplayRouteMapper.mapToUpdateLocation(0.0, location)
+        mapboxNavigation.mapboxReplayer.pushEvents(listOf(event))
+        mapboxNavigation.startReplayTripSession()
+        mapboxNavigation.mapboxReplayer.play()
+    }
+
+    // This may be temporary. We need some way to stop a trip session to further development.
+    // This is here because this class has a reference to MapboxNavigation
+    @OptIn(ExperimentalPreviewMapboxNavigationAPI::class)
+    fun stopTripSession() {
+        mapboxNavigation.stopTripSession()
+        mapboxNavigation.mapboxReplayer.stop()
```

Comment:
Do we want to have a separate `stopSimulatedTripSession` API similar to `startSimulatedTripSession`?

Reply:
I don't know exactly when these need to be called and it's temporary anyway. I'll keep an eye on this going forward.